docker-compose.env.example

```diff
@@ -32,9 +32,9 @@ SECRET_KEY_BASE=changemechangemechangeme"
 # you should comment out or remove the mailhog service from docker-compose.yml,
 # too
 OSEM_EMAIL_ADDRESS="osem@mailhog"
-OSEM_SMTP_AUTHENTICATION="login"
```

Comment:
String quoting is perfectly valid in YAML ( http://www.yaml.org/spec/1.2/spec.html#style/flow/double-quoted )... I don't see where this change (and the next line) are necessary.